docs/source/newsfragments/4341.feature.rst

```diff
@@ -0,0 +1 @@
+The ``Task.join()`` method was changed to attribute :attr:`.Task.join`, which returns a Trigger that will block until the Task is finished, but not return the result. Using the deprecated ``task.join()`` will continue to work in a backwards-compatible way.
```

Comment:
```suggestion
The ``Task.join()`` method was changed to the attribute :attr:`.Task.join`, which returns a Trigger that will block until the Task is finished, but not return the result. Using the deprecated ``task.join()`` will continue to work in a backwards-compatible way.
```
(prevent users reading "attribute" as a verb)

Reply:
This will need to be rewritten for the newer changes anyway.